beluga-mqtt/src/lib.rs

```diff
@@ -264,7 +264,7 @@ async fn poll(
                 }
             }
             Err(conn_err) => {
-                error!(error = %conn_err, "connection error during polling in event loop");
+                error!(error = %conn_err, "connection error during polling");
```

Comment:
I know it can be annoying to type, but tracing has support for `dyn std::error::Error` types:
```suggestion
                error!(error = &conn_err as &dyn std::error::Error, "connection error during polling");
```

The benefit is that this way the formatter actually gets the error, rather than just the string representation. The text logger will for instance add the `source()`: https://docs.rs/tracing-subscriber/latest/src/tracing_subscriber/fmt/format/mod.rs.html#1253-1270